File: deps/src/cftmq/test/sender.cpp
#include "stompc.h"

int main(void)
{
    stomp::connection c;

    if(c.connect("127.0.0.1:40090"))
    {
        if(c.login("root",""))
        {
            for(int i=0;i<10;i++)
            {
                char buf[256]; int n=sprintf(buf,"Hello world %i",i);

//                c.send(buf,"test");

                if(c.send(std::string(buf,n),"test"))
                    printf("%s - ok\n",buf);
                else
                    printf("%s - fail\n",buf);

            }

            c.logout();
        }
    }

    return 0;
}

File: deps/src/cftmq-agent/test/sender.lua
require('luastomp')

con=stomp.connect('127.0.0.1:40090')

if con then

    if con:login('root','') then

--        con:send( { data='hello', destination='test', ['seq-id']='1234', ['chunk-id']='1/2',
--            ['chunk-range']='0-4/10', ['chunk-dgst']='14e1a1785cc9716a6b8ff8161e394337' } )

        con:send( { data='world', destination='test', ['seq-id']='1234', ['chunk-id']='2/2',
            ['chunk-range']='5-9/10', ['chunk-dgst']='14e1a1785cc9716a6b8ff8161e394337' } )

    end

end